src/main/java/org/example/dao/AthleteDao.java

```diff
@@ -0,0 +1,168 @@
+package org.example.dao;
+
+import org.example.entity.Athlete;
+import org.example.entity.City;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AthleteDao {
+
+    private static final String DATABASE_URL = "jdbc:postgresql://localhost:5432/swim_base_2";
+    private static final String DATABASE_USER = "postgres";
+    private static final String DATABASE_PASSWORD = "1313";
+
+
+    public void createAthletes(Athlete athlete) {
```

Comment:
createAthleteS - > createAthlete 
Or just create. This will be called like athletDao.create(..)  - > this will be quite obvious when you see which dao and which method is calling